vandy-login-page/login.php

```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE html>
-<!-- saved from url=(0040)https://sso.vanderbilt.edu/idp/SSO.saml2 -->
+
```

Comment:
Please restore this comment.  
It serves as a breadcrumb in the event this page needs to realign with Vanderbilt's for whatever reason.